tests/image/test_deploy.py

```diff
@@ -135,12 +141,18 @@ def test_progress_verbose_logic(list_mocked, mocked_get, mocked_post, monkeypatc
         result = runner.invoke(
             cli, ["dev", "--version", "test", "--verbose"])
         assert result.exit_code == 0
-        # test for one of the events in the output
-        assert ("{'status': 'progress', 'progress': 100,"
-                " 'total': 100, 'last_step': 'updating panel'}") in result.output
-        # test for result in the end of output
-        assert result.output.endswith("{'status': 'ok', 'project': 1111112,"
-                                      " 'version': 'test', 'spiders': 10}\n")
+        lines = result.output.split('\n')
+        # test that output ends with a newline symbol
+        assert lines[-1] == ''
+        # test that the command succeeded
+        assert _load_deploy_event_result(lines[-2]) == {
+            'status': 'ok', 'project': 1111112, 'version': 'test', 'spiders': 10,
```

Comment:
You could use `DEPLOY_EVENTS_BASE_SAMPLE[-1]` instead of repeating the dict here (and below)